setup_scripts/moab_setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+ 
+#MOAB Installation
+INSTALL_ROUTE=$HOME
+cd $INSTALL_ROUTE
+
+apt install -y autoconf libtool make mpich libblas-dev liblapack-dev libhdf5-dev cmake
```

Comment:
Maybe move all the `apt` calls to a separate script because it needs to be run at `sudo` and the others don't